src/GuzzleHttp/ClientSerializerAdapter.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of the CsaGuzzleBundle package
+ *
+ * (c) Charles Sarrazin <charles@sarraz.in>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+namespace Csa\Bundle\GuzzleBundle\GuzzleHttp;
+
+use GuzzleHttp\Client;
+use Psr\Http\Message\RequestInterface;
+
+class ClientSerializerAdapter extends Client
+{
+    /**
+     * @var string
+     */
+    private $jsonRegex = '((application|text)/([^+]*\+){0,1}json)';
```

Comment:
This won't match with `text/javascript`, which is another content-type which may be used for returning json.